mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -563,14 +590,17 @@ pub enum SyncStatus {
     UpdatingBlocksCache {
         current_scanned_block: u64,
         latest_block: u64,
+        additional_info: Option<String>,
```

Comment:
I think we should leave the actual message to the client/GUIs, we need to provide only info about the first block height and if it's sapling or not. e.g.
```rust
pub struct FirstSyncBlock {
    pub requested: u64,
    pub is_pre_sapling: bool,
    pub actual: u64,
}

pub enum SyncStatus {
    UpdatingBlocksCache {
        first_block: FirstSyncBlock,
        current_scanned_block: u64,
        latest_block: u64,
    },
    BuildingWalletDb {
        first_block: FirstSyncBlock,
        current_scanned_block: u64,
        latest_block: u64,
    },
    TemporaryError(String),
    Finished {
        first_block: FirstSyncBlock,
        block_number: u64,
    },
}
```

Reply:
done thanks

Reply:
It's good to supply the info in order as I showed `first_block` -> `current_scanned_block` -> `latest_block`, this doesn't affect GUIs but it's good for reading on cli, postman, etc..